### ISSUE TITLE
"Element with ID 'paynl_payment_paylink' already exists" since 1.4.4

### ISSUE THREAD

**Issue by cornips** [paynl/magento2-plugin#34]
Installing version 1.4.4 on Magento 2.2.6 with option 'paylink' enabled, gives the following error when trying to get payment options for order:

```
main.CRITICAL: Element with ID 'paynl_payment_paylink' already exists. [] []
main.CRITICAL: Block order.create.billing.method.form throws exception and cannot be rendered. [] []
```

Using version 1.4.3 doesn't give this error with the same configuration.

**Comment by andypieters** [comment]
Thanks for reporting this. i think i know where this comes from.
I added this line in the config: https://github.com/paynl/magento2-plugin/blob/master/etc/config.xml#L12
Because i needed to set a defeult value for discount tax.
In the checkout magento complains that no model is set, so i set the model to paylink. but it appears this causes an error in the admin order.
I'll let you know when there is an update available.

**Comment by pietervanuhm** [comment]
Same error here.

`[2018-11-08 12:39:22] main.CRITICAL: Element with ID 'paynl_payment_paylink' already exists. [] []
[2018-11-08 12:39:22] main.CRITICAL: Block order.create.billing.method.form throws exception and cannot be rendered. [] []`

@andypieters please give a fast solution to fix this?

**Comment by cornips** [comment]
@pietervanuhm fast solution is to downgrade to 1.4.3 😉

**Comment by pietervanuhm** [comment]
i have never downgrade a module, can you help me with that?

**Comment by cornips** [comment]
Sure, just require your composer package with version number defined. I've downgraded tot 1.4.3 with this code:

```
composer require paynl/magento2-plugin:1.4.3
```

**Comment by andypieters** [comment]
An update is on the way, i've already pushed it to the master branch.
I will test everything on a clean magneto installation and release the new version (1.4.5) today

**Comment by pietervanuhm** [comment]
@andypieters do you give a shout here when the new version is published?

**Comment by andypieters** [comment]
I just released version 1.4.5
This fixes the admin order issue

**Comment by andypieters** [comment]
@pietervanuhm SHOUT!
